augur/filter/__init__.py

```diff
@@ -63,11 +64,17 @@ def register_arguments(parser):
     Since priorities represent relative values between strains, these values can be arbitrary.""")
     subsample_group.add_argument('--subsample-seed', type=int, help="random number generator seed to allow reproducible subsampling (with same input data).")
 
-    output_group = parser.add_argument_group("outputs", "possible representations of filtered data (at least one required)")
+    output_group = parser.add_argument_group("outputs", "options related to outputs, at least one of the possible representations of filtered data (--output, --output-metadata, --output-strains) is required")
     output_group.add_argument('--output', '--output-sequences', '-o', help="filtered sequences in FASTA format")
     output_group.add_argument('--output-metadata', help="metadata for strains that passed filters")
     output_group.add_argument('--output-strains', help="list of strains that passed filters (no header)")
     output_group.add_argument('--output-log', help="tab-delimited file with one row for each filtered strain and the reason it was filtered. Keyword arguments used for a given filter are reported in JSON format in a `kwargs` column.")
+    output_group.add_argument(
+        '--empty-results-reporting',
+        type=EmptyResultsReportingMethod,
+        choices=list(EmptyResultsReportingMethod),
+        default=EmptyResultsReportingMethod.ERROR,
+        help="How should empty filter results be reported.")
```

Comment:
_non-blocking_
Related to my minor comment about using "ignore" instead of "skip", I slightly prefer "handled" to "reported" in the context of "how should empty filter results be...".

Reply:
There's precedent for `--*-reporting` options in `augur curate` commands.